index.md

```diff
@@ -77,6 +78,7 @@ list of clients. Here are some I like:
 - [astral.ninja](https://astral.ninja/) - Fork of Branle with different UI & global feed
 - [snort.social](https://snort.social/) - Very simple feed with automatic image-upload
 - [yosup.app](https://yosup.app/) - Mobile-friendly and twitter-like
+- [nostrpy](https://github.com/monty888/nostrpy) - Client, relay and tools in python
```

Comment:
There are so many clients and tools, the purpose here is to link to the best ones that are already live (not WIP).

Reply:
You're right. i proposed this one in particular because most of resources are in javascript, while this is a good "swiss knife" in python. It can be used in different areas where python is used.